kv/txn_coord_sender.go

```diff
@@ -504,8 +503,8 @@ func (tc *TxnCoordSender) sendOne(ctx context.Context, call proto.Call) {
 		// without starting a new call, which is hard to trace. Plus, the
 		// below depends on default configuration.
 		tmpDB, err := client.Open(
-			fmt.Sprintf("//%s?priority=%d",
-				call.Args.Header().User, call.Args.Header().GetUserPriority()),
+			fmt.Sprintf("//?priority=%d",
```

Comment:
I have no idea what the previous thing even did anything. The user part doesn't parse into anything:
http://play.golang.org/p/WesaO55s0k